components/Pagination.js

```diff
@@ -0,0 +1,146 @@
+import { useEffect, useState } from "react";
+import { FaChevronLeft, FaChevronRight } from "react-icons/fa";
+import { BsChevronDoubleLeft, BsChevronDoubleRight } from "react-icons/bs";
```

Comment:
We had an problem with Vercel and reaching the file limit, this was because of the icon imports

Please can you changes these to be the full import for each icon (this will become an import for each icon, so 4 lines)

Reply:
Sure